### PR TITLE
move conditionals from gemspec to gemfile and create matrix in travis…

### PR DESCRIPTION
**Pull request by abMatGit**
WERK IN PREGRESS

### CODE REVIEW

**Comment by abMatGit** [comment]
@zvkemp this will make comply compatible with rails 4 without the gemspec throwing up.

File: Gemfile [hunk 1]
```diff
@@ -5,6 +5,14 @@ source "https://rubygems.org"
 # development dependencies will be added by default to the :development group.
 gemspec
 
+gem 'rails', ENV['RAILS_VERSION'] || '~> 3.2.0'
```
Comment (zvkemp) [inline Gemfile]: can we make this 4 by default?

Comment (abMatGit) [inline Gemfile]: we can do that. we can certainly do that.

**Comment by abMatGit** [comment]
@jdoconnor can you review please?

**Comment by jdoconnor** [comment]
+1

**Pull request merged**